src/gui/attributetable/qgsdualview.cpp

```diff
@@ -110,8 +110,6 @@ void QgsDualView::init( QgsVectorLayer *layer, QgsMapCanvas *mapCanvas, const Qg
 
   // This slows down load of the attribute table heaps and uses loads of memory.
   //mTableView->resizeColumnsToContents();
-
-  mFeatureList->setEditSelection( QgsFeatureIds() << mFeatureListModel->idxToFid( mFeatureListModel->index( 0, 0 ) ) );
```

Comment:
Can you add more details on this change. Are there no side effects?

Reply:
Nope. Checking the blame it seems you added it originally to ensure the first record is selected in the feature list, but now at this point the model is ALWAYS empty, so it no longer does anything (except throw warnings). I've checked and the first entry is still selected, so there must be code elsewhere which handles that now.

Reply:
Ah, good, thanks!